js/flightlog_parser.js

```diff
@@ -562,7 +562,6 @@ var FlightLogParser = function(logData) {
                 that.sysConfig.currentMeterOffset = currentMeterParams[0];
                 that.sysConfig.currentMeterScale = currentMeterParams[1];
             break;
-            case "gyro.scale":
```

Comment:
This is not a duplicate. In some older blackbox files `gyro.scale` was used - this maintains compatibility.